<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>610</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>400</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>零基础餐厅</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#LoginDlg{
border-image:url(:/Image/登录界面.png);

}



</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(116, 224, 191);
border-radius:10%;
color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtname">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10%;
background-color:rgb(255, 255, 255);
color:rgb(200, 193, 197);
padding-left:20%;
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入11位手机号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtpwd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10%;
color:rgb(200, 193, 197);
background-color:rgb(255, 255, 255);
padding-left:20%;
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 178, 214);
border-radius:10%;
color:rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
QPushButton#pushButton:hover{  
background-color:rgb(255, 255, 214);  
}  </string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>185</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(218, 80, 104);
border-radius:10%;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>新用户注册</string>
   </property>
  </widget>
  <widget class="QComboBox" name="model_chosen">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #32435E;   
border-radius: 3px;       
down-arrow{image:url(:/Image/下拉框提示图标.png) ;}

</string>
   </property>
   <item>
    <property name="text">
     <string>用户</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>管理员</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>服务员</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>厨师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>经理</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;华文行楷&quot;;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Restaurant</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
